### PR TITLE
Extended Readme

### PR DESCRIPTION
**Pull request by TimNowaczynski**
These are those 2 small hickups I had during project setup. I did not create a ticket for those README fixes, hope that is okay with you :)

### CODE REVIEW

**Review by d4r1091 — COMMENTED** [review]
Awesome stuff @TimNowaczynski thanks for your contribution!
Can I ask you as part of this PR to also update the README title (now being "Firefox iOS") and the AppStore link with the one for the Ecosia App? `https://apps.apple.com/it/app/ecosia/id670881887`

Thanks!

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Firefox for iOS [![codebeat badge](https://codebeat.co/badges/67e58b6d-bc89-4f22-ba8f-7668a9c15c5a)](https://codebeat.co/projects/github-com-mozilla-firefox-ios) [![codecov](https://codecov.io/gh/mozilla-mobile/firefox-ios/branch/main/graph/badge.svg)](https://codecov.io/gh/mozilla-mobile/firefox-ios/branch/main)
```
Comment (TimNowaczynski) [inline README.md:1]: I removed those codebeat badges as well since they also target the original sources

**Comment by TimNowaczynski** [comment]
@d4r1091 Since I will be gone now till next Tuesday, would you go ahead and merge this if everything is fine? Thanks :)

**Review by d4r1091 — APPROVED** [review]
All good @TimNowaczynski ! Thanks for your contribution!

**Comment by TimNowaczynski** [comment]
@d4r1091 The weird thing is that the test are failing, also tried a re-run with the same results. But I hardly doubt that my readme changes could cause that? Anyways, I'll leave things like they are now for better. We can always merge this later on, I hardly doubt any conflicts 👍

**Comment by d4r1091** [comment]
@TimNowaczynski no worries those tests won't even need to be run as part of that. WIll merge them.

**Comment by TimNowaczynski** [comment]
@d4r1091 Thanks!

**Pull request merged**